task/pluto/0.1/pluto.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pluto
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: Deprecation, Kubernetes, APIVersions
+    tekton.dev/displayName: "Pluto"
```

Comment:
can we have one more annotation over here
`tekton.dev/categories: category name`. You can find the list of categories over [here](https://github.com/tektoncd/hub/pull/283/files)